### DISCUSSION TITLE
혹시 기울임꼴 지원도 예정에 있을까요?

### DISCUSSION THREAD

**Discussion by yunu7067** [orioncactus/pretendard#61] [Q&A] [answered]
웹에서 woff2 가변 글꼴과 폰트 패밀리로 `font-family: Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, 'Helvetica Neue', 'Segoe UI', 'Apple SD Gothic Neo', 'Noto Sans KR', 'Malgun Gothic', 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', sans-serif;` 이렇게 주고 사용했습니다. 

웹에서 기울임꼴을 줄 땐 CSS 속성값으로 `font-style: italic;`이나 `font-style: oblique;`을 사용하는데, 둘 다 적용되지 않는 걸 보면 폰트에 기울임꼴이 포함되지 않은 걸로 보입니다. 

그래서 문의드립니다. 혹시 글꼴에 기울임꼴 지원도 예정에 있을까요?

**Comment by yunu7067** [comment, marked as answer]
정정합니다. Windows에서는 정상적으로 기울임꼴이 되고, macOS에서만 안되네요. 지원하지 않는 CSS 속성이 있는지 확인해보고 오겠습니다.

**Reply by orioncactus** [reply]
안녕하세요! 글꼴 파일로 제공되는 Pretendard 기울임꼴은 한국에서는 주로 사용하지 않는 모양으로 지원을 고려하고 있지 않습니다. 다만 소프트웨어 기능을 통한 기울임꼴 지원은 다른 글꼴과 같이 가능하며, macOS 12.3에서 Safari 15.4 및 Chrome 99.0.4844.83에서 `font-style: italic;` 및 `font-style: oblique;`으로 확인해보았을 때 정상적으로 작동되는 것을 확인했습니다. 문제가 발생하는 페이지가 공개 페이지라면, URL을 알려주시면 확인해보겠습니다.

**Reply by yunu7067** [reply]
[https://orgtest-11324.github.io/](https://orgtest-11324.github.io/)입니다. macOS에서 웨일 브라우저로 테스트했을땐 안됐는데, Safari 및 Chrome에서는 정상적으로 기울임꼴이 적용되네요.... 웨일 브라우저 버그일 가능성이 커보입니다. 번거롭게 해서 죄송합니다😭😭

**Comment by kms0219kms** [comment]
<img width="1552" alt="스크린샷 2022-04-21 오전 7 39 39" src="https://user-images.githubusercontent.com/67222970/164336454-40b7d8e3-d7fc-40e3-b93d-405afcec7005.png">
<img width="1552" alt="스크린샷 2022-04-21 오전 7 38 50" src="https://user-images.githubusercontent.com/67222970/164336471-401f83e4-cb8a-4038-a9b9-9a8e9874ae04.png">

macOS 12.3 (M1)에서 Whale 버전 3.14.133.23으로 테스트했으나 정상 작동되는 것을 확인했습니다.

혹시 최신버전 웨일로 업데이트 후에 시도해 보시겠어요?

**Reply by yunu7067** [reply]
업데이트 후에 기울임꼴 정상적으로 출력되네요! 웨일 문제가 맞았나 봅니다 감사합니다! 😂

**Comment by kms0219kms** [comment]
해결 되셨다니 다행입니다! 더 문의사항 있으시면 다시 이슈 열어주세요!
 
-----Original Message-----
